examples/nlp/language_modeling/prepare_truncated_data.py

```diff
@@ -54,27 +59,215 @@ def _process_line(line, task, tokenizer, max_seq_length, tokens_to_generate_extr
     truncation_seperator = line['truncation_seperator']
     truncation_seperator_tokens_len = len(tokenizer.text_to_tokens(truncation_seperator))
```

Comment:
Since this is now different for each position, we can keep this logic inside the block:
```
if truncation_pos == 'right':
```